samples/dashboard/cypress/add-layout/demo.html

```diff
@@ -0,0 +1,7 @@
+<!-- <script src="https://code.highcharts.com/dashboard.js"></script>
```

Comment:
Remove?

Reply:
I'd assume, this is a default way of importing our modules, and `es-modules` is a current workaround, so I would leave it as it is.